src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/EntityComponent.scala

```diff
@@ -306,16 +306,16 @@ trait EntityComponent {
       val entityIds = entityRecs.map(_.id).toSeq
 
       def insertTempAttributes(): ReadWriteAction[Unit] = {
-        attributeNamespaceQuery.getMap.flatMap { attributeNamespaceMap =>
+        //attributeNamespaceQuery.getMap.flatMap { attributeNamespaceMap =>
```

Comment:
Yes, apparently so!  Increasing the concurrency count to 1000 makes this fail even with the "fix"